index.html

```diff
@@ -47,7 +47,7 @@ <h3>I'am a software developer</h3>
     </section>
```

Comment:
- [ ]  Kindly make sure you stick to the design as much as possible. You may consider fixing the spacing (margin and padding) of these elements.
- [ ] Also you may consider using the same typo, `font` and `color` as the template provided in Figma

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/204845059-e2f17be5-7a53-4dc4-8653-07f32eb2c532.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/204844939-641ef49c-33f2-449d-87c8-74a04e406a5a.png)

Reply:
In the optional suggestion, you said i should use a new repo, since I have submitted this one can I create a new repos and do it again?